docs/ldml/tr35-general.md

```diff
@@ -902,157 +902,160 @@ As with other identifiers in CLDR, the American English spelling is used for uni
-			See also <a href="https://physics.nist.gov/cuu/Units/binary.html">Prefixes for binary multiples</a></li></ul></td></tr>
-
-<tr><td><a name='prefix_component' href='#prefix_component'>prefix_component</a></td><td>:=</td>
-    <td>[a-z]{3,∞}
-        <ul><li><em>Constraint:</em> must be value in: &lt;unitIdComponent type="prefix"&gt;.</li></ul></td></tr>
-
-<tr><td><a name='base_component' href='#base_component'>base_component</a></td><td>:=</td>
-    <td>[a-z]{3,∞}
-        <ul><li><em>Constraint:</em> must not be a value in any of the following:<br>
-			&lt;unitIdComponent type="prefix"&gt;<br>
-			or &lt;unitIdComponent type="suffix"&gt; <br>
-			or &lt;unitIdComponent type="power"&gt;<br>
-			or &lt;unitIdComponent type="and"&gt;<br>
-			or &lt;unitIdComponent type="per"&gt;.
-		</li>
-		<li><em>Constraint:</em> must not have a prefix as an initial segment.</li>
-		<li><em>Constraint:</em> no two different base_components will share the first 8 letters.
-				(<b>For more information, see <a href="#Unit_Identifier_Uniqueness">Unit Identifier Uniqueness</a>.)</b>
-			</li>
-		</ul>
-	</td></tr>
-
-<tr><td><a name='suffix_component' href='#suffix_component'>suffix_component</a></td><td>:=</td>
-    <td>[a-z]{3,∞}
-        <ul>
-			<li><em>Constraint:</em> must be value in: &lt;unitIdComponent type="suffix"&gt;</li>
-		</ul></td></tr>
-
-<tr><td><a name='mixed_unit_identifier' href='#mixed_unit_identifier'></a></td><td>:=</td>
-    <td>(single_unit | pu_single_unit) ("-" and "-" (single_unit | pu_single_unit ))*
-        <ul><li><em>Example: foot-and-inch</em></li>
-		</ul></td></tr>
-
-<tr><td>and</td><td>:=</td>
-    <td>"and"
-		<ul>
-			<li><em>Constraint:</em> The token 'and' is the single value in &lt;unitIdComponent type="and"&gt;</li>
-		</ul></td></tr>
-
-<tr><td><a name='long_unit_identifier' href='#long_unit_identifier'>long_unit_identifier</a></td><td>:=</td>
-    <td>grouping "-" core_unit_identifier</td></tr>
-
-<tr><td>grouping</td><td>:=</td>
-    <td>[a-z]{3,∞}</td></tr>
-
-<tr><td><a name='currency_unit' href='#currency_unit'>currency_unit</a></td><td>:=</td>
-    <td>"curr-" [a-z]{3}
-        <ul>
-			<li><em>Constraint:</em> The first part of the currency_unit is a standard prefix; the second part of the currency unit must be a valid <a href="tr35.md#UnicodeCurrencyIdentifier">Unicode currency identifier</a>.</li>
-		</ul>
-		<ul>
-            <li><em>Examples:</em> <b>curr-eur</b>-per-square-meter, or pound-per-<b>curr-usd</b></li>
-			<li><em>Note:</em> CLDR does not provide conversions for currencies; this is only intended for formatting.
-				The locale data for currencies is supplied in the <code>currencies</code> element, not in the <code>units</code> element.</li>
-        </ul>
-	</td></tr>
-
-</tbody></table>
+<a name="syntax"></a>
+#### Unit Syntax
+
+The formal syntax for identifiers is provided below, in [EBNF](tr35.md#ebnf). 
```

Comment:
"provided below" does not make sense if the link is to a different document.

Reply:
I looked at this. The link on EBNF is a pointer to the documentation on our EBNF, which is applicable to what is below. But the wording is clumsy.

I'll make a PR to change it to 

The formal [EBNF](https://github.com/unicode-org/cldr/blob/main/docs/ldml/tr35.md#ebnf) syntax for identifiers is provided below: